PotreeConverterPartitioned/copier/parallelcopier.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+import multiprocessing
+import shutil
+import subprocess
+from copier import Copier
+
+from ..loggingwrapper import LoggingWrapper
+
+
+class ParallelCopier(Copier):
+    '''Copies files in parallel command'''
+    def fileChunks(self, filesList, nChunks):
```

Comment:
chunks is a misleading name here, as it often implies creating subsets of a single file object. Here it is used to describe subsets of. whol;e files created from a list of files. Not problematic per se, but as the MPI converter code does use chunks in the former definition this could cause confusion.